main.sh

```diff
@@ -27,27 +27,52 @@ Response=$(
     --url "$SERVER_URL/api-pub/v1/axe-watcher/gh/$COMMIT_SHA"
 )
 
-IssueCount=$(echo "$Response" | jq .last_run_violation_count)
+LastRunCreatedAt=$(echo "$Response" | jq -r .last_run_created_at)
+LastRunViolationCount=$(echo "$Response" | jq -r .last_run_violation_count)
+LastRunPageStateCount=$(echo "$Response" | jq -r .last_run_page_state_count)
+LastRunNewViolationCount=$(echo "$Response" | jq -r .last_run_new_violation_count)
+LastRunResolvedViolationCount=$(echo "$Response" | jq -r .last_run_resolved_violation_count)
 IssuesOverA11yThreshold=$(echo "$Response" | jq .issues_over_a11y_threshold)
 AxeURL=$(echo "$Response" | jq -r .axe_url)
 ProjectName=$(echo "$Response" | jq -r .project_name)
+DifferenceInPageStateCount=$(echo "$Response" | jq -r .difference_in_page_states)
 
 # Only set outs when running in GitHub Actions.
 if [ -n "${GITHUB_OUTPUT+x}" ]; then
   echo "project=$ProjectName" >>"$GITHUB_OUTPUT"
   echo "axe_url=$SERVER_URL$AxeURL" >>"$GITHUB_OUTPUT"
-  echo "issue_count=$IssueCount" >>"$GITHUB_OUTPUT"
+  echo "created_at=$LastRunCreatedAt" >>"$GITHUB_OUTPUT"
+  echo "issue_count=$LastRunViolationCount" >>"$GITHUB_OUTPUT"
+  echo "new_issues=$LastRunNewViolationCount" >> "$GITHUB_OUTPUT"
+  echo "resolved_issues=$LastRunResolvedViolationCount" >> "$GITHUB_OUTPUT"
   echo "issues_over_a11y_threshold=$IssuesOverA11yThreshold" >>"$GITHUB_OUTPUT"
+  echo "difference_in_page_states=$DifferenceInPageStateCount" >> "$GITHUB_OUTPUT"
+  echo "page_states=$LastRunPageStateCount" >> "$GITHUB_OUTPUT"
 fi
 
 echo "Project: $ProjectName"
-echo "axe DevHub found $IssueCount accessibility violations."
+echo "axe Developer Hub accessibility results:"
+echo "$LastRunViolationCount violations found"
+echo "$LastRunPageStateCount page states found"
+echo "$LastRunNewViolationCount new issues found"
+echo "$LastRunResolvedViolationCount issues resolved"
+
+existing_issues_count=$(( $LastRunViolationCount - $LastRunNewViolationCount ))
+existing_issues_count=$(( existing_issues_count < 0 ? 0 : existing_issues_count ))
+
+echo "$existing_issues_count previous issues remain"
+
+if [ "$DifferenceInPageStateCount" -gt 0 ]; then
+  echo "$DifferenceInPageStateCount more page states"
+else
+  echo "$DifferenceInPageStateCount fewer page states"
```

Comment:
Should probably use a different message (or no message) for 0, and need to multiply by -1 for this message to make sense with <0

Reply:
Updatee